fpdf/__init__.py

```diff
@@ -27,6 +27,7 @@
 from .html import HTMLMixin, HTML2FPDF
 from .prefs import ViewerPreferences
 from .template import Template, FlexTemplate
+from .text_region import TextColumns
```

Comment:
Why importing this class there if it's not used anywhere, including in `__all__` below?

Reply:
I've removed it from `__all__` earlier, but apparently forgot about the import.